Dockerfile

```diff
@@ -1,9 +1,14 @@
 FROM eclipse-temurin:11-jdk-focal
 
 ARG MAVEN_VERSION=3.8.6
+# https://downloads.apache.org/maven/maven-3/3.8.6/binaries/apache-maven-3.8.6-bin.tar.gz.sha512
+ARG MAVEN_SHA512=f790857f3b1f90ae8d16281f902c689e4f136ebe584aba45e4b1fa66c80cba826d3e0e52fdd04ed44b4c66f6d3fe3584a057c26dfcac544a60b301e6d0f91c26
```

Comment:
Verifying checksums is a "nice to have" goal, but afaik dependabot does not update them, if it updates dependencies.
This could cause a validation issue in the future, if this is not considered.

Reply:
How could it cause a validation issue when we don't use Dependabot to update the Maven version?

Reply:
Don't we in https://github.com/jenkinsci/core-pr-tester/blob/master/.github/dependabot.yml#L8?

Reply:
We do not. That updates the version of the image specified in the `FROM` directive, but it does not parse `ARG` directives whose values look like versions and try to update them (moreover, how could it?).